pycbc/results/pygrb_plotting_utils.py

```diff
@@ -56,11 +54,6 @@
 from matplotlib import rc
```

Comment:
Can you move line 55 into the various executables? This would prevent interactive plots from working in e.g. ipython notebooks if this gets called.

Reply:
You moved line 56! That one's not a problem ```matplotlib.use('agg')``` should be avoided in module code.

Reply:
Oh no, I went to the wrong line in vi I guess.  +54 instead of +55 when opening.  Ugh.

Reply:
Okay, reverted.  Moving `matplotlib.use('agg')` out of the way now.  (Btw, it is also in  pycbc/pycbc/results/followup.py and  pycbc/pycbc/results/scatter_histograms.py)

Reply:
Okay. Thanks for spotting that, those should be moved too, but that's not for this PR!

Reply:
Oh no, I was simply dumping on your todo list :-P

Reply:
RIght, that should be it.  And it was line 53 in the end :-D